src/Views/AppInfoView.vala

```diff
@@ -1218,6 +1202,25 @@ public class AppCenter.Views.AppInfoView : AppCenter.AbstractAppContainer {
         });
     }
 
+    private void origin_setup_factory (Object object) {
+        var title = new Gtk.Label ("") {
+            xalign = 0
+        };
+
+        var list_item = object as Gtk.ListItem;
+        list_item.set_data ("title", title);
+        list_item.set_child (title);
+    }
+
+    private void origin_bind_factory (Object object) {
+        var list_item = object as Gtk.ListItem;
+
+        var package = (AppCenterCore.Package) list_item.get_item ();
+
+        var title = list_item.get_data<Gtk.Label> ("title");
```

Comment:
I think instead of doing this manually you can just cast the `list_item.child` as a label

Reply:
You got it!